xml/depl_nodes.xml

```diff
@@ -7640,6 +7640,28 @@ radosgw_urls:
        No special configuration is needed for &o_comp;. It uses the
        configuration from the &o_blockstore; &barcl; to connect with volumes
        stored in SES.
+       For ephemeral storage, &o_comp; re-uses the <literal>rbd_store_user</literal>
+       and key from &o_blockstore; but has a separate <literal>rbd_store_pool</literal>
+       defined in the SES configuration.
+      </para>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>RadosGW Integration</term>
+     <listitem>
+      <para>
+       Besides &o_blockstore;, SES cluster can also be used as a &o_objstore;
+       replacement for object storage. If <literal>radosgw_urls</literal> section is present
+       in uploaded SES configuration, first of the URLs are registered
+       in the &o_ident; catalog as "&swift;"/"object-store" service. Some
```

Comment:
what's the difference between `&swift;` and `&o_objstore;`?

Reply:
You can see all of the entitys here: https://github.com/SUSE-Cloud/doc-cloud/blob/maintenance/cloud_9/xml/entity-decl.ent#L76

Reply:
thx!